files/mods.json

```diff
@@ -1980,6 +1980,27 @@
         "forge_id": "crashhelper",
         "hash": "1662607fdad6fa24d7abfce4aeb3b6c9"
     },
+    {
+        "id": "arrowpath",
+        "display": "Arrow Path",
+        "creator": "Aycy",
+        "description": "Visual trajectory of arrows.",
+        "icon": "aycy.jpg",
+        "actions": [
+            {
+                "icon": "youtube.png",
+                "text": "Showcase",
+                "link": "https://www.youtube.com/watch?v=f3iP-lzobxk"
+            }
+        ],
+        "categories": [
+            "5;All PvP",
+            "6;Recommended PvP"
```

Comment:
sus

Reply:
this should also be removed from recommend